cmd/beaker/dataset.go

```diff
@@ -299,6 +286,64 @@ func newDatasetRenameCommand() *cobra.Command {
 	}
 }
 
+func newDatasetSizeCommand() *cobra.Command {
+	return &cobra.Command{
+		Use:   "size <dataset> <prefix?>",
```

Comment:
Square brackets are idiomatic for "optional".
```suggestion
		Use:   "size <dataset> [prefix]",
```

Reply:
Looks like this appears in a few places.  In VSCode, regex replace: `<(\w+)\?>` -> `[$1]`

Reply:
Fixed